ches/emailHandler.ts

```diff
@@ -5,19 +5,23 @@ import { NewRequestTemplate } from '@/emails/templates/NewRequestTemplate';
 import { RequestApprovalTemplate } from '@/emails/templates/RequestApprovalTemplate';
 import { RequestRejectionTemplate } from '@/emails/templates/RequestRejectionTemplate';
 import { adminEmails } from './emailConstant';
-import chesService from './index';
+import { sendEmail } from './index';
 
 export const sendNewRequestEmails = async (formData: PrivateCloudCreateRequestBody) => {
   const email = render(NewRequestTemplate({ formData }), { pretty: true });
   try {
-    const send1 = chesService.send({
+    const send1 = await sendEmail({
       body: email,
       // For all project contacts. Sent when the project set deletion request is successfully submitted
-      to: [formData.projectOwner.email, formData.primaryTechnicalLead.email, formData.secondaryTechnicalLead?.email],
+      to: [
+        formData.projectOwner.email,
+        formData.primaryTechnicalLead.email,
+        formData.secondaryTechnicalLead?.email,
+      ].filter(Boolean),
```

Comment:
this `filter` is unnecessary in favor of `safeEmails` function inside send function https://github.com/bcgov/platform-services-registry/pull/1306/files#diff-47dbc6dc1d6ea3dc3cd61dd971bc213868f28f32ef8d3618d38b1585ad98981eR90